ada/ada

```diff
@@ -283,7 +283,8 @@ while [ $# -gt 0 ] ; do
       usage
       ;;
     --version )
-      git describe --tags --abbrev=0
+      _VERSION_PLACEHOLDER=$(git describe --tags --abbrev=0 2>/dev/null || echo "unknown")
```

Comment:
```suggestion
      _VERSION_PLACEHOLDER=$(git describe --tags --abbrev=0 2>/dev/null || echo "v2.1")
```

Reply:
Thanks! Good solution. Let's then set the version manually in case the script is not in a git repo.

Reply:
Totally fine by me :-)

Reply:
Thinking a little longer about it, i prefer your solution. i.e. not manually setting it in ada, but let the ansible script do it.